docs/window.dox

```diff
@@ -1009,6 +1013,10 @@ Hidden windows can be shown with @ref glfwShowWindow.
 glfwShowWindow(window);
 @endcode
 
+By default, this function will also set the input focus to that window. Set
+the [GLFW_FOCUS_ON_SHOW](@ref GLFW_FOCUS_ON_SHOW_hint) window hint to change
+this behavior for all newly created windows, or change the behaviour for an
```

Comment:
@dougbinks I think GLFW is using American English, so you may want to change `behaviour` to `behavior`.

Reply:
Yes, that should be behaviour to keep consistency. I've updated with the latest push.

Reply:
I think your auto-correct is failing you :)